mmv1/products/workbench/Instance.yaml

```diff
@@ -385,6 +385,26 @@ properties:
               values:
                 - VIRTIO_NET
                 - GVNIC
+            - !ruby/object:Api::Type::Array
+              name: accessConfigs
+              immutable: true
+              description: |
+                Optional. An array of configurations for this interface. Currently, only one access
+                config, ONE_TO_ONE_NAT, is supported. If no accessConfigs specified, the
+                instance will have an external internet access through an ephemeral
+                external IP address.
+              item_type: !ruby/object:Api::Type::NestedObject
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: externalIp
+                    default_from_api: true
```

Comment:
Can we make this field as required, since this is the only field in the block, we'll try to prevent users from sending empty blocks if possible.

Reply:
Cannot make this required. If specified, this will be a static ip. If not, an ephemeral ip will be assigned.

Reply:
Does it mean that sending `access_configs {}` is different from not sending the block to the API?

Reply:
It is not different. Made the suggested change.